test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,7 +9,35 @@
 import org.junit.Test;
 
 public class UtilsTest {
+    @Test
+    public void isAnyNull() throws Exception {
+        // only one null
+        assertSomeNull((Object) null);
+
+        // only one non null
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+
+        // multi, no null
+        assertNoNull("abc", "abc");
+        assertNoNull("abc", "", "abc", "ABC");
+        assertNoNull("", "abc", "a", "abc");
+        assertNoNull(1, Integer.valueOf(1));
+
+        // multi, some null
+        assertSomeNull(null, 1, Integer.valueOf(1));
+        assertSomeNull(null, null);
+        assertSomeNull(null, "a", "b", null);
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
```

Comment:
It's good that you followed the style used by the code nearby.
However, in this case, there is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method. Asserting the result of the method itself is fine.
The existing test uses assertAreUnique() in order to prevent Arrays.asList(..) from being used each time.